aiohttp/client.py

```diff
@@ -187,7 +188,7 @@ def __init__(self, *, connector: Optional[BaseConnector]=None,
                  version: HttpVersion=http.HttpVersion11,
                  cookie_jar: Optional[AbstractCookieJar]=None,
                  connector_owner: bool=True,
-                 raise_for_status: bool=False,
+                 raise_for_status: Union[bool, Callable[[ClientResponse], None]]=False,  # noqa
```

Comment:
I'm curious why `None` is accepted here?
In `ClientSession.request()` `None` means *take session default* but for session constructor itself `None` is invalid value from my understanding.

Reply:
It's not, `None` is the return type from the callable.

Reply:
Sorry, my fault.